cmd/api/config.go

```diff
@@ -28,8 +28,9 @@ type config struct {
 		MaxRequestPerInterval uint64 `default:"10"`
 	}
 	Gateway struct {
-		ExternalURIPrefix   string `default:"https://testnet.tableland.network"`
-		MetadataRendererURI string `default:""`
+		ExternalURIPrefix    string `default:"https://testnet.tableland.network"`
+		MetadataRendererURI  string `default:""`
+		AnimationRendererURI string `default:""`
```

Comment:
Although `AnimationRendererURI ` and `MetadataRendererURI` point to the same server, I added a new config because in the future they will not.